src/debugger.fs

```diff
@@ -41,10 +41,11 @@ module Debugger =
                          hostname = "remotedev.io"
                          port = 443
                          secure = true
+                         name = "Elmish"
```

Comment:
Magic string "Elmish" used in several places. Here, L141, and L163. Would empty string be a better default?

Reply:
Well one is for fallback the other is the default. I am not sure, what is best here though.